src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,73 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private static final ColorSupplier color = new ColorSupplier();
+    private static final Random random = new Random();
+    private static final FigureType[] figures = FigureType.values();
+    private static final double RANDOM_MAX_VALUE = 5.1;
+    private static final double DEFAULT_CIRCLE_RADIUS = 10;
+    private static final String DEFAULT_CIRCLE_COLOR = Color.WHITE.name();
```

Comment:
please do not create an additional field for this value. Use it directory where needed